src/main/java/org/fcrepo/spec/testsuite/versioning/LdprmHttpOptions.java

```diff
@@ -88,33 +109,41 @@ public void ldprmOptionsMaySupportDelete(final String uri) {
                                         ps);
 
         //create an LDPRm
+        final String mementoURI = createMemento(uri, info);
+        final Response mementoResponse = doGet(mementoURI);
         //verify that a subsequent OPTIONS return "DELETE"
+        confirmPresenceOfHeaderValueInMultiValueHeader("Allow", "DELETE", mementoResponse);
+
     }
 
     /**
      * 4.2.3
      *
      * @param uri The repository root URI
      */
-    //@Test(groups = {"MUST"})
+    @Test(groups = {"MUST"})
     @Parameters({"param1"})
     public void ldprmMustNotSupportPost(final String uri) {
         final TestInfo info = setupTest("4.2.3", "ldprmMustNotSupportPost",
                                         "An LDPRm must not support POST",
                                         "https://fcrepo.github.io/fcrepo-specification/#ldprm-post",
                                         ps);
-
         //create an LDPRm
+        final String mementoURI = createMemento(uri, info);
+        final Response mementoResponse = doGet(mementoURI);
         //verify that a subsequent OPTIONS does not return POST
+        confirmAbsenceOfHeaderValueInMultiValueHeader("Allow", "POST", mementoResponse);
         //verify that issuing a POST returns a 405 (method not allowed).
+        final Response response = doPostUnverified(mementoURI);
+        assertEquals(response.getStatusCode(), 405, "Response to a POST should be 405");
```

Comment:
The 405 seems to be implementation specific  - perhaps it's what's people would use, but I'm not sure if it should be assumed.  Thoughts?

Reply:
It's not specified in the spec, so perhaps this should just check for a 4XX error?

Reply:
@bseeger : I'm not sure either.   It seems to me that anything other than 405 would indicate some other problem.  I would suggest that we keep it as 405 and change it if we run into complaints from one or more Fedora implementers.

Reply:
I agree that 405 is the appropriate response here. 
As a point of reference for other tests that need to support 4xx status codes, there is this helper:
https://github.com/fcrepo/Fedora-API-Test-Suite/blob/master/src/main/java/org/fcrepo/spec/testsuite/AbstractTest.java#L360